components/expenses/MarkExpenseAsPaidBtn.js

```diff
@@ -65,13 +65,14 @@ class MarkExpenseAsPaidBtn extends React.Component {
     } catch (e) {
       const error = getErrorFromGraphqlException(e).message;
       this.setState({ error, loading: false });
```

Comment:
Do we still need to store the `error ` in state?

Reply:
Doesn't appear so. I'll deal with that.